src/style.css

```
@@ -0,0 +1,121 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 600;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  --form-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  --form-shadow-hover: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search-form_big {
+  width: 100%;
+  height: 70px;
+}
+
+.search-form_small {
+  width: 100%;
+  height: 42px;
+}
```

```suggestion
.search-form_small {
  height: 42px;
}
```
width already settled for big form
check similar cases for other elements.